web/src/styles/app/table.css

```diff
@@ -33,7 +33,13 @@
 
 .table th,
 .table td {
-  border-top: 1px solid #e7e7e7;
+  border-top: 1px solid;
+  border-color: #e7e7e7;
+}
+
+.table th.border-none,
+.table td.border-none {
+  border: 0;
```

Comment:
There's a `border-none` atomic class, but because we have `.table th/td` styles, those were winning on specificity, so I added this one to win ever better...  Ugh?  😄

Reply:
ahahah, the joy of CSS. It's like the Joy of Painting, but with fewer happy little trees and more flustered developers. (Also, sounds like the existing rules are probably too greedy, which I'll keep in mind when I have to tackle table styling.)

Reply:
🌳